### PR TITLE
Fix fan rotation

### PR DESCRIPTION
**Pull request by cdce8p**
The fan rotation attributes should be `forward` and `reverse` instead of `left` and `right`. Fix for #1158.
If asked for I can rebase this PR againt the `polymer-3-modulize` branch.

### CODE REVIEW

**Comment by cdce8p** [comment]
On hold until #1154 is merged.
I will rebase it then.

**Comment by cdce8p** [comment]
I rebased the PR.
Should be ready to be merged now.

**Review by balloob — APPROVED** [review]

**Pull request merged**
